online-shop/src/components/ProductListView/ProductList/ProductList.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { ProductDTOArray } from '../../../model/model'
+import { ProductItem } from "./ProductItem/ProductItem";
+import './ProductList.scss';
+export { ProductList };
+
+const ProductList: React.FC<ProductDTOArray> = (productArray: ProductDTOArray) => {
+
+    const productItems = productArray.products.map((product) => {
+        var url = "/products/" + product.id;
+        return <ProductItem key={product.id} category={product.category} name={product.name} price={product.price} 
```

Comment:
FYI: you can also use the spread operator if an object (`product` in this case) has the same properties and the react component's props. E.g. you could write this as: `<ProductItem key={product.id} {...product} />`.